sources/docker-compose.yml

```diff
@@ -235,6 +235,8 @@ services:
       interval: 5s
       timeout: 5s
       retries: 5
+    networks:
+      - local-network
```

Comment:
👍 

Just want to confirm (for my own understanding), adding this will ensure that clamAV container is able to communicate with the other containers in the same `local-network` ?

Reply:
Yes, that's correct.